corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js

```diff
@@ -756,6 +756,7 @@ hqDefine('app_manager/js/detail-screen-config.js', function () {
                         property_name: that.field,
                         multimedia: that.screen.config.multimedia,
                         values_are_icons: that.original.format == 'enum-image',
+                        values_are_conditions: that.original.format == 'conditional-enum'
```

Comment:
Expected '===' and instead saw '=='. (eqeqeq)
Missing trailing comma. (comma-dangle)